coz

```diff
@@ -185,6 +188,10 @@ _run_parser.add_argument('--fixed-speedup',
                          type=int, choices=list(range(0, 101)), default=None,
                          help='Evaluate optimizations of a specific amount')
 
+_run_parser.add_argument('--debug-libraries',
```

Comment:
I'm not sure I see a point to adding a flag for this when users can just use `LD_DEBUG=libs` directly when running coz. Can you remove the changes to this file?